Minor improvements

D4nte opened the pull request:
Small changes, mainly regarding MarkDown syntax, I fixed while reading the specs.

D4nte committed:
Fix indentation and close bracket

As per #266

D4nte committed:
Clarify what message is used for updates

D4nte committed:
Add protocol to ensure IDE understand it as an URL and not local path

D4nte committed:
WakuSub is deprecated, WakuRelay should be used

As per #266

D4nte committed:
Correct table align syntax to make it valid

At least 3 dashes `-` are needed to be valid markdown syntax and a new
line must be present before the table.

D4nte committed:
Special character are removed from anchors by MD

oskarth reviewed (approved):
Thanks!

jm-clius commented on specs/waku/v2/waku-v2-rpc-api.md:
```diff
@@ -189,26 +192,29 @@ The following structured types are defined for use on the Store API:
+| ----: | :---: | :---: |----------- |
 | `messages` | `Array`[[`WakuMessage`](#WakuMessage)] | mandatory | Array of retrieved historical messages |
-| `pagingOptions` | [`PagingOptions`](#PagingOptions) | [conditional](#`get_waku_v2_store_v1_messages`) | Paging information from which to resume further historical queries |
+| `pagingOptions` | [`PagingOptions`](#PagingOptions) | [conditional](#get_waku_v2_filter_v1_messages) | Paging information from which to resume further historical queries |
```
```suggestion
| `pagingOptions` | [`PagingOptions`](#PagingOptions) | [conditional](#get_waku_v2_store_v1_messages) | Paging information from which to resume further historical queries |
```
Ah, thanks, I see this link was broken due to the backticks, but it should indeed point to `get_waku_v2_store_v1_messages` where the conditions for including `pagingOptions` are discussed.

jm-clius reviewed (approved):
Added suggestion re link to `get_waku_v2_store_v1_messages` vs `get_waku_v2_filter_v1_messages`, but otherwise lgtm!

D4nte committed:
Correct link

Conditions for including `pagingOption` are discussed in
`get_waku_v2_store_v1_messages`

D4nte commented:
I can see that the spell check failed in all merged PRs so I'll ignore it.
